app/javascript/tests/patient/vaccines/VaccineModal.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { shallow } from 'enzyme';
+import VaccineModal from '../../../components/patient/vaccines/VaccineModal';
+
+function getShallowWrapper() {
+  return shallow(
+    // TODO
+    <VaccineModal 
+    />
+  );
+}
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('VaccineModal', () => {
+  it('TODO', () => {
```

Comment:
Are you planning on writing these before you go?  If not, I'd say remove the files and create a JIRA to write them specifically for these two components

Reply:
That was originally the plan but time has slipped away so yes I made a ticket to address. I'm honestly okay with keeping them for now as skeleton files unless you're strongly opposed. Ticket is [here](https://tracker.codev.mitre.org/browse/SARAALERT-1379).

Reply:
In the past, we haven't kept any skeleton files to reduce confusion.  Since there isn't anything substantial in here (new mocks, any rendering tests, etc.), I'd say just remove it because who knows when we'll get to that ticket